src/htcp.cc

```diff
@@ -1086,14 +1088,20 @@ htcpHandleTstResponse(htcpDataHeader * hdr, char *buf, int sz, Ip::Address &from
             return;
         }
 
-        if ((t = d->resp_hdrs))
-            htcpReply.hdr.parse(t, d->respHdrsSz);
+        if ((t = d->resp_hdrs)) {
+            Http::ContentLengthInterpreter interpreter;
+            htcpReply.hdr.parse(t, d->respHdrsSz, interpreter);
```

Comment:
Similar to HttpRequest::parseHeader(), we could add HtcpReply::parseHeader() to simplify this code.